frontend/src/component/project/Project/ProjectSettings/ChangeRequestConfiguration/ChangeRequestConfiguration.tsx

```diff
@@ -0,0 +1,158 @@
+                            inputProps={{ 'aria-label': original.environment }}
+                            onClick={onClick(original.environment)}
+                        />
+                    </Box>
+                ),
+                width: 100,
+                disableGlobalFilter: true,
+                disableSortBy: true,
+            },
+        ],
+        []
+    );
+
+    const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
+        useTable(
+            {
+                columns,
+                data,
+                sortTypes,
+                autoResetGlobalFilter: false,
+                disableSortRemove: true,
+                defaultColumn: {
+                    Cell: TextCell,
+                },
+            },
+            useGlobalFilter
+        );
+
+    return (
+        <PageContent
+            header={<PageHeader titleElement="Change request configuration" />}
+            // isLoading={loading}
+        >
+            <Alert severity="info" sx={{ mb: 3 }}>
+                If change request is enabled for an environment, then any change
+                in that environment needs to be approved before it will be
+                applied
+            </Alert>
+
+            <Table {...getTableProps()}>
+                <SortableTableHeader
+                    headerGroups={headerGroups as HeaderGroup<object>[]}
+                />
+                <TableBody {...getTableBodyProps()}>
+                    {rows.map(row => {
+                        prepareRow(row);
+                        return (
+                            <TableRow hover {...row.getRowProps()}>
+                                {row.cells.map(cell => (
+                                    <TableCell {...cell.getCellProps()}>
+                                        {cell.render('Cell')}
+                                    </TableCell>
+                                ))}
+                            </TableRow>
+                        );
+                    })}
+                </TableBody>
+            </Table>
+
+            <Dialogue
+                onClick={() => {
+                    alert('clicked');
+                    /* FIXME: API action */
+                }}
+                open={dialogState.isOpen}
+                onClose={() =>
+                    setDialogState(state => ({ ...state, isOpen: false }))
+                }
+                primaryButtonText="Enable"
+                secondaryButtonText="Cancel"
+                title="Enable change request"
+            >
+                <Typography>
+                    You are about to enable “Change request”
+                    {dialogState.environment
+                        ? ` for ${dialogState.environment}`
+                        : ''}
+                    .
+                </Typography>
+                <Typography>
+                    When enabling change request for an environment, you need to
```

Comment:
in the design there was a picture with the change request flow. are we deliberately skipping it for now?

Reply:
Yes. Designs changed recently. I'll update it in next PR.